lua/yaml_nvim/init.lua

```diff
@@ -57,15 +58,20 @@ local yank = function(key, value, register)
 end
 
 M.view = function()
-	assure_yaml_filetype(function()
+	vim.notify(M.get())
+end
+
+M.get = function()
```

Comment:
`get` is too generic and arguably meaningless. Have you considered a more precise name for that function?

```suggestion
M.get_yaml_key_and_value = function()
```